sdk/monitor/azure-monitor-query/azure/monitor/query/_helpers.py

```diff
@@ -118,7 +118,7 @@ def native_col_type(col_type, value):
     if col_type == "datetime":
         try:
             value = Deserializer.deserialize_iso(value)
-        except: # pylint: disable=bare-except
+        except Exception:
```

Comment:
You may get a "pylint exception too broad", but this one is fine to disable in this context

Reply:
fixed